### PR TITLE
Add Lock iOS: Logout

### PR DESCRIPTION
**Pull request by lobo**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/libraries/lock-ios/index.md [hunk 1]
```diff
@@ -335,6 +335,35 @@ After a successful sign-up, the user can be logged in automatically using the `l
 
 If you want to show a disclaimer for your app, you will need to set `signUpDisclaimerView`. This view will appear at the bottom of the sign-up screen.
 
+### Logout
+
+If desired, you have ability to let the user logout. For this you need to call `clearSessions` for `A0Lock`.
```
Comment (DylanMeeus) [inline articles/libraries/lock-ios/index.md]: Small fix: "If desired, you have _the_ ability to let the user logout"

File: articles/libraries/lock-ios/index.md [hunk 2]
```diff
@@ -335,6 +335,35 @@ After a successful sign-up, the user can be logged in automatically using the `l
...
 If you want to show a disclaimer for your app, you will need to set `signUpDisclaimerView`. This view will appear at the bottom of the sign-up screen.
 
+### Logout
+
+If desired, you have ability to let the user logout. For this you need to call `clearSessions` for `A0Lock`. 
+This method removes all stored sessions of any IdP in your application.
+
+#### Important notes:
+
+* If the user is logged in using Safari, those sessions will not be cleaned.
```
Comment (DylanMeeus) [inline articles/libraries/lock-ios/index.md]: Suggestion: 
cleaned - cleared 
line underneath: clean them - clear them

(The method is also called _clearSessions_, not _cleanSessions_) :-)

File: articles/libraries/lock-ios/index.md (hunk 1 above)
Comment (lobo) [inline articles/libraries/lock-ios/index.md]: Fantastic, changing now, thanks!

File: articles/libraries/lock-ios/index.md (hunk 2 above)
Comment (lobo) [inline articles/libraries/lock-ios/index.md]: Fantastic, changing now, thanks!

**Pull request merged**